plugins/env/app/models/environment_variable.rb

```diff
@@ -64,13 +67,19 @@ def serialize(variables, env_scopes)
       end.join("\n")
     end
 
-    def config_service_folder(project, required:)
-      url = ENV["CONFIG_SERVICE_URL"]
-      raise KeyError, "CONFIG_SERVICE_URL not set" if required && !url
-      "#{url}/samson/#{project.permalink}"
+    def config_service_folder(project)
```

Comment:
this is called with `required: false` from the UI ... best boot the app and see if it works/makes sense

Reply:
Ahhh I found it.  Didn't realize this was getting called from an ERB template.